### PR TITLE
refactor: Remove xwf from pipelined

### PR DESCRIPTION
**Pull request by alexzurbonsen**
## Summary

This PR relates to https://github.com/magma/magma/issues/13546. It removes xwf functionality from pipelined.

## Test Plan

- CI
- Integ Tests (https://github.com/alexzurbonsen/magma/actions/runs/2876430498)

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/python/magma/pipelined/app/egress.py [hunk 1]
```diff
@@ -140,10 +140,6 @@ def _get_default_flow_msgs(self, datapath) -> DefaultMsgsMap:
     def initialize_on_connect(self, datapath):
         self._datapath = datapath
         self._setup_non_nat_monitoring()
-        # TODO possibly investigate stateless XWF(no sessiond)
-        if self.config.setup_type == 'XWF':
-            self.delete_all_flows(datapath)
-            self._install_default_flows(datapath)
```
Comment (nstng) [inline lte/gateway/python/magma/pipelined/app/egress.py:146]: is `_install_default_flows` now unused?

File: lte/gateway/python/magma/pipelined/app/ingress.py [hunk 2]
```diff
@@ -276,10 +276,6 @@ def finish_init(self, _):
 
     def initialize_on_connect(self, datapath):
         self._datapath = datapath
-        # TODO possibly investigate stateless XWF(no sessiond)
-        if self.config.setup_type == 'XWF':
-            self.delete_all_flows(datapath)
-            self._install_default_flows(datapath)
```
Comment (nstng) [inline lte/gateway/python/magma/pipelined/app/ingress.py:282]: is `_install_default_flows` now unused?

File: lte/gateway/python/magma/pipelined/app/middle.py [hunk 3]
```diff
@@ -180,10 +180,6 @@ def finish_init(self, _):
 
     def initialize_on_connect(self, datapath):
         self._datapath = datapath
-        # TODO possibly investigate stateless XWF(no sessiond)
-        if self.config.setup_type == 'XWF':
-            self.delete_all_flows(datapath)
-            self._install_default_flows(datapath)
```
Comment (nstng) [inline lte/gateway/python/magma/pipelined/app/middle.py:186]: is `_install_default_flows` now unused?

**Review by nstng — CHANGES_REQUESTED** [review]

**Comment by nstng** [comment]
`lte/gateway/python/magma/pipelined/tests/snapshots/test_inout.InOutTestXWF.testFlowSnapshotMatch.snapshot` should be obsolete now.

File: lte/gateway/python/magma/pipelined/app/egress.py (hunk 1 above)
Comment (alexzurbonsen) [reply lte/gateway/python/magma/pipelined/app/egress.py:146]: Yes, your are right. There are no callers within the class anymore.

File: lte/gateway/python/magma/pipelined/app/ingress.py (hunk 2 above)
Comment (alexzurbonsen) [reply lte/gateway/python/magma/pipelined/app/ingress.py:282]: same as above

File: lte/gateway/python/magma/pipelined/app/middle.py (hunk 3 above)
Comment (alexzurbonsen) [reply lte/gateway/python/magma/pipelined/app/middle.py:186]: same as above

**Comment by alexzurbonsen** [comment]
> `lte/gateway/python/magma/pipelined/tests/snapshots/test_inout.InOutTestXWF.testFlowSnapshotMatch.snapshot` should be obsolete now.

Deleted it.

**Comment by alexzurbonsen** [comment]
A little explanation for egress.py, ingress.py and middle.py:

Since the `_install_default_flows` method is deleted in all three files and `_get_default_flow_msgs` and `_wait_for_responses` do not have any other callers (exception: in egress.py's `EgressController` the method `_wait_for_responses` is still used).

Thus, one might think they should be deleted. `_get_default_flow_msgs` implements an abstract method from `RestartMixin`. One thus might prefer an implementation with empty return value.
But both methods are actually used in `handle_restart` in `RestartMixin`, even though `_wait_for_responses` is not defined as an abstract method.
Since `handle_restart` may be called from some outside caller, it is probably not sensible to mess with it.
  
Accordingly, `_get_default_flow_msgs` and `_wait_for_responses` are not deleted here.

**Review by nstng — APPROVED** [review]
lgtm - run sudo tests locally (/)

**Pull request merged**